pip/resolve.py

```diff
@@ -154,6 +154,11 @@ def resolve(self, requirement_set):
             any(req.has_hash_options for req in root_reqs)
         )
 
+        # Display where finder is looking for packages
+        locations = self.finder.get_formatted_locations()
+        if locations:
+            logger.debug(locations)
```

Comment:
@pfmoore This is currently logging as debug.

Should I make it info, since the only time it would be shown is when the index is not PyPI? It gets buried in the debug mode logging.